promql/bench_test.go

```diff
@@ -41,25 +41,31 @@ func setupRangeQueryTestData(stor *teststorage.TestStorage, _ *promql.Engine, in
 	// representing the number of series each metric name contains.
 	// Metric a_X and b_X are simple metrics where h_X is a histogram.
 	// These metrics will have data for all test time range
-	metrics = append(metrics, labels.FromStrings("__name__", "a_one"))
-	metrics = append(metrics, labels.FromStrings("__name__", "b_one"))
+	metrics = append(metrics,
+		labels.FromStrings("__name__", "a_one"),
+		labels.FromStrings("__name__", "b_one"),
+	)
```

Comment:
This seems worse - two lines longer, and breaks the pattern of succeeding lines.